renovate.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "prConcurrentLimit": 2,
+  "schedule": "every weekend",
```

Comment:
Looks like this is the same schedule we were on for testpilot, I was taking care of those prs for a while each week and Monday was a good time for them 👍